bluebrain/repo-bluebrain/packages/sim-model/package.py

```diff
@@ -146,7 +146,7 @@ def install(self, spec, prefix, install_src=True):
 
         bin/ <- special and special-core
         lib/ <- hoc, mod and lib*mech*.so
-        share/ <- neuron & coreneuron mod.c's (modc and modc_core)
+        share/ <- neuron & coreneuron mod.cpp's (modcpp and modcpp_core)
```

Comment:
```suggestion
        share/ <- neuron & coreneuron mod cpp files (modcpp and modcpp_core)
```
I'm left wondering what `mod.cpp` could possess.